### ISSUE TITLE
CREATE TABLE IF NOT EXISTS doesn't seem to work the 2nd time

### ISSUE THREAD

**Issue by longnguyen11288** [2ndQuadrant/bdr#135]
```
CREATE TABLE IF NOT EXISTS credentials (
        id text,
        uri text,
        access_key text,
        secret_key text,
        PRIMARY KEY (id)
        );
```
Works the first time and 2nd time I'm getting 

```
NOTICE:  relation "credentials" already exists, skipping
ERROR:  trigger "truncate_trigger" for relation "credentials" already exists
CONTEXT:  SQL statement "CREATE TRIGGER truncate_trigger AFTER TRUNCATE ON test.credentials FOR EACH STATEMENT EXECUTE PROCEDURE bdr.queue_truncate()”
```

**Comment by cweibel** [comment]
I see the same thing for 0.9.2.0

**Comment by ringerc** [comment]
This should be fixed in 0.10.0 where we use `tgisinternal` triggers. I don't intend to backport that change to 0.9.x as it's a heavier change than I'm comfortable with in a point release.

(Do not just update to 0.10.0-pre git master, as the schemas etc might change before release and it's a work in progress).

A workaround in 0.9.x would need to use a different approach. We don't have an `IF NOT EXISTS` for triggers, so the simplest option is out.
